app/labor/mailchimp_bot.rb

```diff
@@ -170,4 +188,8 @@ def target_md5_email
     email = saved_changes["unconfirmed_email"] ? saved_changes["email"][0] : user.email
     md5_email(email)
   end
+
+  def previously_subcribed?(error)
+    error.title.match?(/Member In Compliance State/)
```

Comment:
Once we confirmed resubscribing via "pending" is indeed what we want, we can revisit this check and maybe come up with something smarter.